.github/workflows/ci.yml

```diff
@@ -1531,6 +1535,9 @@ jobs:
             xtensa-intel_tgl_adsp_zephyr-elf)
               PLATFORM_ARGS+="-p intel_adsp_cavs25 "
               ;;
+            xtensa-mt8195_adsp_zephyr-elf)
+              PLATFORM_ARGS+="-p intel_adsp_cavs25 "
```

Comment:
Will need to update this after the board config has been merged.

Reply:
Oops, yeah, that's a cut/paste error for sure.  Will fix.